gson/pom.xml

```diff
@@ -7,6 +7,7 @@
     <version>2.8.1-SNAPSHOT</version>
   </parent>
 
+  <packaging>bundle</packaging>
```

Comment:
It still spits out a jar file, but the MANIFEST.MF file will have more entries in it, making this thing a real OSGi bundle.  So, yes, the size changes, somewhat.